cluster.go

```diff
@@ -319,6 +319,11 @@ func (c *Cluster) Topo() ClusterTopo {
 func (c *Cluster) getTopo(p Client) (ClusterTopo, error) {
 	var tt ClusterTopo
 	err := p.Do(Cmd(&tt, "CLUSTER", "SLOTS"))
+	if tt == nil && err == nil {
```

Comment:
Let's make this `len(tt) == 0`

Reply:
Sure ! Thought of that earlier. Let me change this to:
```len(tt) == 0```